robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
Yeah, as we will get completely different set of data that cannot be received in another way. Doing search seems as very redundant and time consuming task for that particular helper